apps/weshipit/pages/qr.tsx

```diff
@@ -2,14 +2,19 @@ import { Text } from '@weshipit/ui';
 import { Layout } from '../components/layout';
 import Script from 'next/script';
 import { linksApi } from './api/links';
+import type { Metadata } from 'next';
+
+export const metadata: Metadata = {
+  title: 'FREE Pair Programming Session',
```

Comment:
instead of duplicating the content you can use `metadata.title` and `metadata.description`.
add a description please.